Update metricflow-commands.md (#5390)
fishtown-analytics/docs.getdbt.com: website/docs/docs/build/metricflow-commands.md

- [`tutorial`](#tutorial) &mdash; Dedicated MetricFlow tutorial to help get you started.
-->

:::tip Run dbt parse to reflect metric changes

> When you make changes to metrics, make sure you run `dbt parse` at a minimum to update the dbt Semantic Layer. This updates the `semantic_manifest.json` file is updated, reflecting your changes when you query metrics. Running `dbt parse` ensures you don't have to rebuild all the models.

Reviewer: ```suggestion
When you make changes to metrics, make sure to run `dbt parse` at a minimum to update the dbt Semantic Layer. This updates the `semantic_manifest.json` file, reflecting your changes when querying metrics. By running `dbt parse`, you won't need to rebuild all the models.
```

Author: oo great catch!
